test/manual/amp-animation/scrollbound-embeds.amp.html

```diff
@@ -14,7 +14,7 @@
     // WARNING: This script tag makes this page an INVALID AMP Page.
     // This code is here for demo purposes and enables the required experiments
     // automatically for this demo so that visitors do not need to enable them manually.
-    var experiments = ['amp-animation', 'amp-position-observer'];
+    var experiments = ['amp-animation', 'amp-position-observer', 'inabox-position-api'];
```

Comment:
@aghassemi Is the scrollbound animation already supported. Basically with the new refractor we need to toggle on `inabox-position-api` to enable position observer in inabox. Let me know if I should remove the experiment flag in this PR

Reply:
scrollbound is still under an experiment flag but will be removed soon. if you are okay with removing `inabox-position-api` experiment, go for it. otherwise mention it in https://github.com/ampproject/amphtml/issues/10875 so they get removed at the same time

Reply:
I'll mention there : )